### ISSUE TITLE
Jupyterlab not showing up in "home" tab in Anaconda 2.1.1

### ISSUE THREAD

**Issue by vpvinc** [ContinuumIO/anaconda-issues#12739]
### Actual Behavior

After updating to Anaconda 2.1.1, jupyterlab is not showing up in the home tab of any of my virtual environments(see screenshot). I can still launch it via conda but it's less convenient than with previous versions. 
Additional notes:
-  jupyterlab IS showing up in the home tab of root environment 
-  jupyter is showing up in the home tab of all environments as in previous version of Anaconda

### Expected Behavior

Jupyterlab app should be available in the home tab of all virtual environments

### Steps to Reproduce

update to Anaconda 2.1.1

##### Anaconda or Miniconda version:

Anaconda 2.1.1

##### Operating System:

windows 10 Home 
Version 10.0.19043 Build 19043

##### `conda info`

(general) C:\Users\VP>conda info

     active environment : general
    active env location : C:\Users\VP\anaconda3\envs\general
            shell level : 1
       user config file : C:\Users\VP\.condarc
 populated config files : C:\Users\VP\.condarc
          conda version : 4.11.0
    conda-build version : 3.21.4
         python version : 3.8.8.final.0
       virtual packages : __cuda=11.0=0
                          __win=0=0
                          __archspec=1=x86_64
       base environment : C:\Users\VP\anaconda3  (writable)
      conda av data dir : C:\Users\VP\anaconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\VP\anaconda3\pkgs
                          C:\Users\VP\.conda\pkgs
                          C:\Users\VP\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\VP\anaconda3\envs
                          C:\Users\VP\.conda\envs
                          C:\Users\VP\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.11.0 requests/2.25.1 CPython/3.8.8 Windows/10 Windows/10.0.19041
          administrator : False
             netrc file : None
           offline mode : False

##### `conda list --show-channel-urls`

(general) C:\Users\VP>conda list --show-channel-urls
 packages in environment at C:\Users\VP\anaconda3\envs\general:

 Name                    Version                   Build  Channel
anyio                     3.4.0            py37h03978a9_0    conda-forge
argcomplete               1.12.3             pyhd8ed1ab_2    conda-forge
argon2-cffi               21.1.0           py37hcc03f2d_2    conda-forge
asgiref                   3.4.1              pyhd8ed1ab_0    conda-forge
async_generator           1.10                       py_0    conda-forge
attrs                     21.2.0             pyhd8ed1ab_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
brotlipy                  0.7.0           py37hcc03f2d_1003    conda-forge
ca-certificates           2020.10.14                    0    anaconda
category_encoders         2.3.0              pyhd8ed1ab_0    conda-forge
certifi                   2020.6.20                py37_0    anaconda
cffi                      1.15.0           py37hd8e9650_0    conda-forge
charset-normalizer        2.0.8              pyhd8ed1ab_0    conda-forge
click                     8.0.3            py37h03978a9_1    conda-forge
cloudpickle               2.0.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
console_shortcut          0.1.1                         4    defaults
cryptography              36.0.0           py37h65266a2_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
debugpy                   1.5.1            py37hf2a7229_0    conda-forge
decorator                 5.1.0              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
entrypoints               0.3             pyhd8ed1ab_1003    conda-forge
fastapi                   0.70.0             pyhd8ed1ab_0    conda-forge
freetype                  2.10.4               h546665d_1    conda-forge
greenlet                  1.1.2            py37hf2a7229_1    conda-forge
h11                       0.12.0             pyhd8ed1ab_0    conda-forge
icu                       68.2                 h0e60522_0    conda-forge
idna                      3.1                pyhd3deb0d_0    conda-forge
imbalanced-learn          0.8.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        4.8.2            py37h03978a9_0    conda-forge
importlib_metadata        4.8.2                hd8ed1ab_0    conda-forge
importlib_resources       5.4.0              pyhd8ed1ab_0    conda-forge
intel-openmp              2021.4.0          h57928b3_3556    conda-forge
ipykernel                 6.5.0            py37h4038f58_1    conda-forge
ipython                   7.30.0           py37h03978a9_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jbig                      2.1               h8d14728_2003    conda-forge
jedi                      0.18.1           py37h03978a9_0    conda-forge
jinja2                    3.0.3              pyhd8ed1ab_0    conda-forge
joblib                    1.1.0              pyhd8ed1ab_0    conda-forge
jpeg                      9d                   h8ffe710_0    conda-forge
json5                     0.9.5              pyh9f0ad1d_0    conda-forge
jsonschema                4.2.1              pyhd8ed1ab_0    conda-forge
jupyter_client            7.1.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.9.1            py37h03978a9_1    conda-forge
jupyter_server            1.12.1             pyhd8ed1ab_0    conda-forge
jupyterlab                3.2.4              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_server         2.8.2              pyhd8ed1ab_0    conda-forge
kiwisolver                1.3.2            py37h8c56517_1    conda-forge
lcms2                     2.12                 h2a16943_0    conda-forge
lerc                      3.0                  h0e60522_0    conda-forge
libblas                   3.9.0              12_win64_mkl    conda-forge
libcblas                  3.9.0              12_win64_mkl    conda-forge
libclang                  11.1.0          default_h5c34c98_1    conda-forge
libdeflate                1.8                  h8ffe710_0    conda-forge
liblapack                 3.9.0              12_win64_mkl    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libtiff                   4.3.0                hd413186_2    conda-forge
libzlib                   1.2.11            h8ffe710_1013    conda-forge
lightgbm                  3.2.1            py37hf2a7229_0    conda-forge
llvmlite                  0.36.0           py37habb0c8c_0    conda-forge
lz4-c                     1.9.3                h8ffe710_1    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markupsafe                2.0.1            py37hcc03f2d_1    conda-forge
matplotlib                3.4.3            py37h03978a9_1    conda-forge
matplotlib-base           3.4.3            py37h4a79c79_2    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mistune                   0.8.4           py37hcc03f2d_1005    conda-forge
mkl                       2021.4.0           h0e2418a_729    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
nbclassic                 0.3.4              pyhd8ed1ab_0    conda-forge
nbclient                  0.5.9              pyhd8ed1ab_0    conda-forge
nbconvert                 6.3.0            py37h03978a9_1    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.5.1              pyhd8ed1ab_0    conda-forge
notebook                  6.4.6              pyha770c72_0    conda-forge
numba                     0.53.0           py37h49d82ab_0    conda-forge
numpy                     1.21.4           py37h940b05c_0    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openjpeg                  2.4.0                hb211442_1    conda-forge
openssl                   1.1.1l               h8ffe710_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.3.3            py37h9386db6_0    conda-forge
pandoc                    2.16.2               h8ffe710_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parso                     0.8.2              pyhd8ed1ab_0    conda-forge
patsy                     0.5.2              pyhd8ed1ab_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    8.4.0            py37hd7d9ad0_0    conda-forge
pip                       21.3.1             pyhd8ed1ab_0    conda-forge
plotly                    5.3.1              pyhd8ed1ab_0    conda-forge
prometheus_client         0.12.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.22             pyha770c72_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pydantic                  1.8.2            py37hcc03f2d_2    conda-forge
pygments                  2.10.0             pyhd8ed1ab_0    conda-forge
pynndescent               0.5.5              pyh6c4a22f_0    conda-forge
pyopenssl                 21.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.6              pyhd8ed1ab_0    conda-forge
pyqt                      5.12.3           py37h03978a9_8    conda-forge
pyqt-impl                 5.12.3           py37hf2a7229_8    conda-forge
pyqt5-sip                 4.19.18          py37hf2a7229_8    conda-forge
pyqtchart                 5.12             py37hf2a7229_8    conda-forge
pyqtwebengine             5.12.1           py37hf2a7229_8    conda-forge
pyrsistent                0.18.0           py37hcc03f2d_0    conda-forge
pysocks                   1.7.1            py37h03978a9_4    conda-forge
python                    3.7.10          h7840368_102_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-multipart          0.0.5                      py_0    conda-forge
python_abi                3.7                     2_cp37m    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pywin32                   302              py37hcc03f2d_2    conda-forge
pywinpty                  1.1.6            py37h7f67f24_0    conda-forge
pyzmq                     22.3.0           py37hcce574b_1    conda-forge
qt                        5.12.9               h5909a2a_4    conda-forge
requests                  2.26.0             pyhd8ed1ab_1    conda-forge
scikit-learn              1.0              py37ha78be43_1    conda-forge
scipy                     1.7.3            py37hb6553fb_0    conda-forge
seaborn                   0.11.2               hd8ed1ab_0    conda-forge
seaborn-base              0.11.2             pyhd8ed1ab_0    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                59.4.0           py37h03978a9_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sniffio                   1.2.0            py37h03978a9_2    conda-forge
sqlalchemy                1.4.27           py37hcc03f2d_0    conda-forge
sqlite                    3.37.0               h8ffe710_0    conda-forge
starlette                 0.16.0             pyhd8ed1ab_0    conda-forge
statsmodels               0.13.1           py37hec80d1f_0    conda-forge
tbb                       2021.4.0             h2d74725_1    conda-forge
tenacity                  8.0.1              pyhd8ed1ab_0    conda-forge
terminado                 0.12.1           py37h03978a9_1    conda-forge
testpath                  0.5.0              pyhd8ed1ab_0    conda-forge
threadpoolctl             3.0.0              pyh8a188c0_0    conda-forge
tk                        8.6.11               h8ffe710_1    conda-forge
tornado                   6.1              py37hcc03f2d_2    conda-forge
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
typing-extensions         4.0.0                hd8ed1ab_0    conda-forge
typing_extensions         4.0.0              pyha770c72_0    conda-forge
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
umap-learn                0.5.1            py37h03978a9_1    conda-forge
urllib3                   1.26.7             pyhd8ed1ab_0    conda-forge
uvicorn                   0.15.0           py37h03978a9_2    conda-forge
vc                        14.2                 hb210afc_5    conda-forge
vs2015_runtime            14.29.30037          h902a5da_5    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          1.2.1            py37h03978a9_0    conda-forge
wheel                     0.37.0             pyhd8ed1ab_1    conda-forge
win_inet_pton             1.1.0            py37h03978a9_3    conda-forge
winpty                    0.4.3                         4    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.6.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h8ffe710_1013    conda-forge
zstd                      1.5.0                h6255e5f_0    conda-forge

#Screenshot of home tab without jupyterlab:
![Capture](https://user-images.githubusercontent.com/74609847/144396653-4157ae17-ea0a-4d4b-9091-d152ff0e0370.PNG)

**Comment by grandqwere** [comment]
It helped me:
- delete jupyterlab
- reinstall jupyterlab

**Comment by csoja** [comment]
I was not able to reproduce your problem with Navigator 2.1.1, I do see JupyterLab on my home tab.  It looks like you are getting jupyterlab from conda-forge. it may not be built correctly for Navigator to pick it up for the home screen?  what happens when you install jupyterlab from repo.anaconda.com?  If you scroll down, is JupyterLab listed farther down? (I cannot tell from your screenshot as j comes after g)

**Comment by vpvinc** [comment]
@csoja jupyterlab is not listed further down but you are right. I uninstalled jupyterlab and reinstalled it again using `conda install -n jupyterlab` and it got installed through conda-forge again, so I used `conda install -n jupyterlab -c defaults jupyterlab` instead and it solved the issue.
Now It looks like conda checks conda-forge first instead of defaults if you do not use the -c flag. I do not know how to correct that. For information conda-forge was added to channels in anaconda (see screenshot)
![image](https://user-images.githubusercontent.com/74609847/145415044-50359c0c-72f3-43b3-a11b-c38833f23fea.png)

**Comment by csoja** [comment]
Great - I'm glad you've figured it out.  You may want to add an issue on the conda-forge feedstock for jupyterlab that it is not being seen as an application in Navigator.  I'm going to close this issue now though, as it looks like Navigator and the default packages are working as expected.
